ui/tests/acceptance/secrets/backend/engines-test.js

```diff
@@ -0,0 +1,44 @@
+import { currentRouteName } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import mountSecrets from 'vault/tests/pages/settings/mount-secret-backend';
+import withFlash from 'vault/tests/helpers/with-flash';
+import backendsPage from 'vault/tests/pages/secrets/backends';
+import authPage from 'vault/tests/pages/auth';
+
+module('Acceptance | engine/disable', function(hooks) {
+  setupApplicationTest(hooks);
+
+  hooks.beforeEach(function() {
+    return authPage.login();
+  });
+
+  test('disable engine', async function(assert) {
+    // first mount an engine so we can disable it.
+    let enginePath = `alicloud-${new Date().getTime()}`;
+    await mountSecrets.visit();
+    await mountSecrets.selectType('alicloud');
+    await withFlash(
+      mountSecrets
+        .next()
+        .path(enginePath)
+        .submit()
+    );
+
```

Comment:
I think lines 19-26 could be replaced here with:
`await mountSecrets.enable('alicloud', enginePath);`
(Sorry couldn't figure out multi-line suggestions)

Reply:
oooh, didn't notice that in the test page. thank you!